src/libopensc/pkcs15-coolkey.c

```diff
@@ -186,7 +186,7 @@ static int
 coolkey_get_attribute_bytes(sc_card_t *card, sc_cardctl_coolkey_object_t *obj, CK_ATTRIBUTE_TYPE type, u8 *data, size_t *data_len, size_t max_data_len)
 {
 	const u8 *val;
```

Comment:
this case does not ring the "maybe uninitialized errors" for you?

Reply:
No, sorry, this doesn't.